src/components/ArrowKeyFocusManager.js

```diff
@@ -92,6 +53,15 @@ class ArrowKeyFocusManager extends Component {
         );
     }
 
+    componentDidUpdate(prevProps) {
+        if (prevProps.maxIndex === this.props.maxIndex) {
+            return;
+        }
+        if (this.props.focusedIndex > this.props.maxIndex) {
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/20152
This has caused a small regression, where selecting max number of participants in a list would focus the first participant
(when we select max number of participants, we hide other options from the list, which updates maxIndex, making `focusedIndex` greater than `maxIndex` and calling `this.onArrowDownKey()`)
We've added a `shouldResetIndexOnEndReached` prop to disable this behavior on `Invite` And `NewGroup` pages